bin.src/generateCommands.py

```diff
@@ -0,0 +1,581 @@
+#!/usr/bin/env python
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import argparse
+import os
+import re
+import shlex
+import sys
+import textwrap
+
+import yaml
+
+from typing import Any, Dict, Iterator, List, Mapping, Optional, Sequence, Tuple
+
+
+def main():
+    parser = argparse.ArgumentParser(description="""
+    Process a yaml file and generate the desired calibrations
+    """)
+
+    parser.add_argument('dataDir', type=str, help="Root of data repository")
```

Comment:
I'm replacing `'` to `"`. I never type `'` for myself because `'` (SHIFT+7) is far less accessible than `"` (SHIFT+2) in a jp106 keyboard. Occurrences of `'` were almost surely copied from somewhere.